test_conformance/math_brute_force/main.cpp

```diff
@@ -539,6 +561,8 @@ static void PrintUsage(void)
     vlog("\t\t-d\tToggle double precision testing. (Default: on iff khr_fp_64 "
          "on)\n");
     vlog("\t\t-f\tToggle float precision testing. (Default: on)\n");
+    vlog("\t\t-g\tToggle half precision testing. (Default: on if khr_fp_16 "
```

Comment:
I tried to run just the fp16 tests for acos with
`./test_conformance/math_brute_force/test_bruteforce acos -d -f  -w -v`, i.e disabling float and double testing. However, float testing still seems to be run in the output.
 

```
acos...
1:            acos fp32  rlxbase:    4026531840 step:  16777216 scale:       256 buf_elements:     65536 ulps:4.000 ThreadCount: 8
base:    3758096384 step:  16777216 scale:       256 buf_elements:     65536 ulps:4.000 ThreadCount: 8
base:    3489660928 step:  16777216 scale:       256 buf_elements:     65536 ulps:4.000 ThreadCount: 8
base:    3221225472 step:  16777216 scale:       256 buf_elements:     65536 ulps:4.000 ThreadCount: 8
base:    2952790016 step:  16777216 scale:       256 buf_elements:     65536 ulps:4.000 ThreadCount: 8
base:    2684354560 step:  16777216 scale:       256 buf_elements:     65536 ulps:4.000 ThreadCount: 8
base:    2415919104 step:  16777216 scale:       256 buf_elements:     65536 ulps:4.000 ThreadCount: 8
base:    2147483648 step:  16777216 scale:       256 buf_elements:     65536 ulps:4.000 ThreadCount: 8
base:    1879048192 step:  16777216 scale:       256 buf_elements:     65536 ulps:4.000 ThreadCount: 8
base:    1610612736 step:  16777216 scale:       256 buf_elements:     65536 ulps:4.000 ThreadCount: 8
base:    1342177280 step:  16777216 scale:       256 buf_elements:     65536 ulps:4.000 ThreadCount: 8
base:    1073741824 step:  16777216 scale:       256 buf_elements:     65536 ulps:4.000 ThreadCount: 8
base:     805306368 step:  16777216 scale:       256 buf_elements:     65536 ulps:4.000 ThreadCount: 8
base:     536870912 step:  16777216 scale:       256 buf_elements:     65536 ulps:4.000 ThreadCount: 8
base:     268435456 step:  16777216 scale:       256 buf_elements:     65536 ulps:4.000 ThreadCount: 8
base:             0 step:  16777216 scale:       256 buf_elements:     65536 ulps:4.000 ThreadCount: 8
Wimp pass           3.37 @ 0x1.2b5p-1
2:            acos fp16     base:    4026531840 step:  16777216 scale:       128 buf_elements:    131072 ulps:4.000 ThreadCount: 8
base:    3758096384 step:  16777216 scale:       128 buf_elements:    131072 ulps:4.000 ThreadCount: 8
base:    3489660928 step:  16777216 scale:       128 buf_elements:    131072 ulps:4.000 ThreadCount: 8
base:    3221225472 step:  16777216 scale:       128 buf_elements:    131072 ulps:4.000 ThreadCount: 8
base:    2952790016 step:  16777216 scale:       128 buf_elements:    131072 ulps:4.000 ThreadCount: 8
base:    2684354560 step:  16777216 scale:       128 buf_elements:    131072 ulps:4.000 ThreadCount: 8
base:    2415919104 step:  16777216 scale:       128 buf_elements:    131072 ulps:4.000 ThreadCount: 8
base:    2147483648 step:  16777216 scale:       128 buf_elements:    131072 ulps:4.000 ThreadCount: 8
base:    1879048192 step:  16777216 scale:       128 buf_elements:    131072 ulps:4.000 ThreadCount: 8
base:    1610612736 step:  16777216 scale:       128 buf_elements:    131072 ulps:4.000 ThreadCount: 8
base:    1342177280 step:  16777216 scale:       128 buf_elements:    131072 ulps:4.000 ThreadCount: 8
base:    1073741824 step:  16777216 scale:       128 buf_elements:    131072 ulps:4.000 ThreadCount: 8
base:     805306368 step:  16777216 scale:       128 buf_elements:    131072 ulps:4.000 ThreadCount: 8
base:     536870912 step:  16777216 scale:       128 buf_elements:    131072 ulps:4.000 ThreadCount: 8
base:     268435456 step:  16777216 scale:       128 buf_elements:    131072 ulps:4.000 ThreadCount: 8
base:             0 step:  16777216 scale:       128 buf_elements:    131072 ulps:4.000 ThreadCount: 8
Wimp pass           1.33 @ 0x1p-1
acos passed
PASSED 2 of 2 sub-tests.
PASSED test.
ThreadPool: thread 2 exiting.
ThreadPool: thread 1 exiting.
ThreadPool: thread 3 exiting.
ThreadPool: thread 5 exiting.
ThreadPool: thread 6 exiting.
ThreadPool: thread 7 exiting.
ThreadPool: thread 4 exiting.
ThreadPool: thread 0 exiting.
Thread pool exited in a orderly fashion.
*** ResumeSleep() is not implemented on this platform.
```